.github/workflows/build.yml

```diff
@@ -68,31 +68,12 @@ jobs:
       working-directory: ${{env.GITHUB_WORKSPACE}}
       # Add additional options to the MSBuild command line here (like platform or verbosity level).
       # See https://docs.microsoft.com/visualstudio/msbuild/msbuild-command-line-reference
-      run: msbuild /m /p:Configuration=${{env.BUILD_CONFIGURATION}} /p:Platform=${{env.BUILD_PLATFORM}} /p:Analysis='True' ${{env.SOLUTION_FILE_PATH}}
+      run: msbuild /m /p:Configuration=${{env.BUILD_CONFIGURATION}} /p:Platform=${{env.BUILD_PLATFORM}} ${{env.SOLUTION_FILE_PATH}}
+      # run: msbuild /m /p:Configuration=${{env.BUILD_CONFIGURATION}} /p:Platform=${{env.BUILD_PLATFORM}} /p:Analysis='True' ${{env.SOLUTION_FILE_PATH}}
```

Comment:
This "optimizes" the build by removing the Analysis='True' protection.   What do we give up by doing that?   Do we need something else (e.g., a daily build) to compensate?

Reply:
draft PR. Temporarily removed analysis flag to make verification quick. This will not be checked in as is.

Reply:
Its a tradeoff between spinning 2 extra github runners for doing essentially duplicate tasks or sacrifice 5 minutes extra per PR.
Note that this is an experimental PR and I am not convinced if I am invoking the actions correctly. I have to run the experiments on the main repo as all the selfhosted runners have the token for this repo and I cannot run the experiment from my fork. So please bear with me while I run these experiments :-)